### PR TITLE
eris chains/services cat; closes #43

### LINKED ISSUE #43: `services cat` // `chains cat`
should just `cat` the service definition file. perhaps ran through a colorizer.

### PR DESCRIPTION
**Pull request by zramsay**

### CODE REVIEW

File: commands/chains.go [hunk 1]
```diff
@@ -266,6 +267,17 @@ var chainsGraduate = &cobra.Command{
 	},
 }
 
+var chainsCat = &cobra.Command{
+	Use:   "cat [name]",
+	Short: "Displays service file.",
```
Comment (compleatang) [inline commands/chains.go]: `Displays a chain definition file` Similar change in 275.

File: commands/services.go [hunk 2]
```diff
@@ -248,6 +249,17 @@ Use the --force flag to also remove the service definition file.`,
 	},
 }
 
+var servicesCat = &cobra.Command{
+	Use:   "cat [name]",
+	Short: "Displays service file.",
```
Comment (compleatang) [inline commands/services.go]: service `definition` file <- anal stuff I know, but these helps are important to get right.

File: services/manage.go [hunk 3]
```diff
@@ -267,6 +268,16 @@ func RmServiceRaw(servNames []string, containerNumber int, force, rmData bool) e
 	return nil
 }
 
+func CatServiceRaw(servName string, containerNumber int, ops *def.ServiceOperation) error {
+
+	cat, err := ioutil.ReadFile(path.Join(ServicesPath, servName+".toml"))
+	if err != nil {
+		return err
```
Comment (compleatang) [inline services/manage.go]: definition files are allowed in ".yaml", "yml", "json" and toml formats.

**Comment by compleatang** [comment]
nice! Great stuff.

File: services/manage.go (hunk 3 above)
Comment (zramsay) [inline services/manage.go]: should it be restricted to only those formats?

Comment (compleatang) [inline services/manage.go]: Yes. cause we'll have other files and stuff in there. And those (for now) are the only types which our config lib (spf13/viper) reads.

**Pull request merged**